src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.health = health;
```

Comment:
```suggestion
  constructor(name) {
    this.health = 100;
```
health is not passed anywhere

Reply:
All instances of the `Animal` class must have health and a name.
Health is 100 by default. - This discription from task. Can I not consider it in solution?